docs/network_firewall.rst

```diff
@@ -163,7 +163,7 @@ Connect to the pfSense WebGUI
    .. note:: The *Unsafe Browser* is, as the name suggests, **unsafe**
 	     (its traffic is not routed through Tor). However, it is
 	     the only option because Tails `intentionally disables LAN
-	     access`_ in the **Tor Browser**.
+	     access` in the **Tor Browser**.
```

Comment:
@freddymartinez9 hey Freddy! the trailing underscore is a feature of reStructuredText. You might want to read up on the syntax of .rst if you want to contribute to our documentation: http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html

This was a hyperlink, and this change actually breaks it. If you go further down, to about line 197, you'll see where the hyperlink target was defined:

```
.. _intentionally disables LAN access: https://labs.riseup.net/code/issues/7976
```

Just letting you know!